tests/lib/rules/no-extra-parens.js

```diff
@@ -459,13 +488,15 @@ ruleTester.run("no-extra-parens", rule, {
         // ["all", { enforceForArrowConditionals: false }]
         { code: "var a = b => 1 ? 2 : 3", options: ["all", { enforceForArrowConditionals: false }] },
         { code: "var a = (b) => (1 ? 2 : 3)", options: ["all", { enforceForArrowConditionals: false }] },
+        { code: "var a = (b) => ((1 ? 2 : 3))", options: ["all", { enforceForArrowConditionals: false }] },
```

Comment:
@kaicataldo 
> Just to double check I'm understanding all these test changes correctly, this is the only test covering the code change in this PR, correct? All the others look to be covering existing behavior.

This case is also covering the change because it warned in the actual version. -[Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLWV4dHJhLXBhcmVuczogW1wiZXJyb3JcIiwgXCJhbGxcIiwgeyBcImVuZm9yY2VGb3JBcnJvd0NvbmRpdGlvbmFsc1wiOiBmYWxzZSB9XSAqL1xuXG5jb25zdCBmID0gYyA9PiAoKDEgPyAyIDogMykpOyAvLyBVbm5lY2Vzc2FyeSBwYXJlbnRoZXNlcyBleHByZXNzaW9uLiIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6MTEsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnsianN4Ijp0cnVlfX0sInJ1bGVzIjp7fSwiZW52Ijp7ImJyb3dzZXIiOmZhbHNlfX19)